src/coreclr/md/heaps/external.h

```diff
@@ -16,6 +16,6 @@
 #include "../external.h"
 #include "../export.h"
 
-#include <stgpool.h>
+#include "../inc/stgpool.h"
```

Comment:
Can this be just `#include "stgpool.h"` ? Please consider doing that everywhere under src\coreclr\md